Workbooks/WorkbooksMetadata.json

```diff
@@ -479,5 +479,18 @@
     "templateRelativePath": "OneIdentity.json",
     "subtitle": "",
     "provider": "One Identity LLC.​"
-  }
+  },
+  {
+    "workbookKey": "AzureSentinelSecurityAlerts",
+    "logoFileName": "Azure_Sentinel.svg",
+    "description": "Security Alerts dashboard for alerts in your Azure Sentinel environment.",
+    "dataTypesDependencies": [ "SecurityAlerts" ],

```

Comment:
I think the relevant data type is "SecurityAlert" and not "SecurityAlert**s**", right?